blueflood-core/src/test/java/com/rackspacecloud/blueflood/rollup/GranularityTest.java

```diff
@@ -22,31 +22,91 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.Calendar;
 import java.util.HashMap;
-import java.util.Iterator;
 import java.util.Map;
 
 public class GranularityTest {
 
-    final long baseMillis = 1335820166000L;
+    //final long baseMillis = 1335820166000L;
+    final long fromBaseMillis = Calendar.getInstance().getTimeInMillis();
+    final long toBaseMillis = fromBaseMillis+ 604800000;
+
+    //An old timestamp that signifies ttld out data.
+    // 619200000 is 8 days, just 1 day more than 7 days ie ttl limit of full gran
+    final long oldToBaseMillis_FullGran = fromBaseMillis - 619200000;
```

Comment:
Minor Nit:  I don't love putting "_" in variable names unless they are all caps.  Is there a reason why you are doing that?  It looks a bit odd to me.